types/context.go

```diff
@@ -170,6 +174,12 @@ func (c Context) SigningValidators() []abci.SigningValidator {
 func (c Context) GasMeter() GasMeter {
 	return c.Value(contextKeyGasMeter).(GasMeter)
 }
+func (c Context) IsCheckTx() bool {
```

Comment:
Can we add spacing between these functions?

Reply:
Happy to add spacing between `IsCheckTx()` and `WithMultiStore()`. Both `GasMeter()` and `IsCheckTx()` are getters, plus file's current coding convention seems having no spaces between functions.

Reply:
Yeah might as well add spacing to all of them 👍

Reply:
Done